packages/relayer/indexer/config.go

```diff
@@ -40,6 +40,7 @@ type Config struct {
 	SyncMode                            SyncMode
 	WatchMode                           WatchMode
 	NumLatestBlocksToIgnoreWhenCrawling uint64
+	TargetBlock                         *uint64
```

Comment:
Use uint64 and just check `== 0` can be more simplier.

Reply:
It won't work . ```== 0``` will set ```TargetBlock``` to 0 and the indexer will only process block 0.